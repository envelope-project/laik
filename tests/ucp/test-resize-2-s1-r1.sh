#!/bin/sh
timeout() { perl -e 'alarm shift; exec @ARGV' "$@"; }
export UCP_COMMANDS_FILE=/home/ge96hoy2/laik/tests/ucp/test-resize-2-s1-r1-commands.txt
timeout 2 ./ucprun.sh --strict-node 0 -n 2 -s 1 ../../examples/resize 1 | LC_ALL='C' sort > test-resize-2-s1-r1.out
cmp test-resize-2-s1-r1.out "$(dirname -- "${0}")/test-resize-2-s1-r1.expected"
