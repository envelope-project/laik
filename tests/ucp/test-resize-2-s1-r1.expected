Epoch 0 / Phase 0: Hello from process 0 of 2
Epoch 0 / Phase 0: Hello from process 1 of 2
Epoch 1 / Phase 1: Hello from process 0 of 2
Epoch 1 / Phase 1: Hello from process 1 of 2
